consensus/parlia/parlia.go

```diff
@@ -938,16 +941,18 @@ func (p *Parlia) distributeFinalityReward(chain consensus.ChainHeaderReader, sta
 	currentHeight := header.Number.Uint64()
 	epoch := p.config.Epoch
 	chainConfig := chain.Config()
-	headhash := header.ParentHash
-	head := chain.GetHeaderByHash(headhash)
 	if currentHeight%epoch != 0 || !chainConfig.IsBoneh(new(big.Int).Sub(header.Number, big.NewInt(1))) {
```

Comment:
if currentHeight%epoch != 0 is ok, no need to judge !chainConfig.IsBoneh(new(big.Int).Sub(header.Number, big.NewInt(1))